src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,92 @@
+import "./Timer.css";
+import React, { useRef, useState } from "react";
+import { Duration } from "./Duration";
+import { InputField } from "../InputField";
+import { TimeEntry } from "./TimeEntry";
+
+export const Timer = () => {
+  //state, what I can set or write
+  //getter a setter
+  // setter set it
+
+  //actual time
+  const [nowTime, setNowTime] = useState(null);
+  // start the timer
+  const [startTime, setStartTime] = useState(null);
+
+  const actualTimer = nowTime - startTime;
+
+  const [listTime, setListTime] = useState([]);
+  console.log(listTime);
+
+  // reference of interal - constant
+  const intervalRef = useRef(null);
+
+  const handleStart = () => {
+    //chci aktulni cas, kdy zacla
+    setStartTime(Date.now());
+
+    setNowTime(Date.now());
```

Comment:
FYI: In this very context, this approach works. But in a context you would need `startTime` and `nowTime` would be the equal (on millisecond), it is better to first store `Date.now()` into constant and then use the constant.